stable/goldilocks/templates/dashboard-deployment.yaml

```
@@ -74,11 +74,11 @@ spec:
               protocol: TCP
           livenessProbe:
             httpGet:
-              path: {{.Values.dashboard.basePath | default "/" }}health
+              path: {{ include "goldilocks.basePath" . }}/health
```

```suggestion
              path: {{ trimSuffix "/" .Values.dashboard.basePath }}/health
```

I think this is simpler (no need for the template). It's also closer to the current logic (which expects a `/` prefix for this value)